### PR TITLE
add tests for gmodules and move to Experimental

### PR DESCRIPTION
**Pull request by fieker**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -28,14 +28,16 @@ include("Rings/FunctionField-test.jl")
...
 end
 
 include("Rings/binomial-ideals-test.jl")
 
-include("Examples/galois-test.jl")
-include("Examples/ModStdQt-test.jl")
-include("Examples/ModStdNF-test.jl")
+include("Experimental/PlaneCurve-test.jl")
+include("Experimental/galois-test.jl")
+include("Experimental/gmodule-test.jl")
```
Comment (fingolfin) [inline test/runtests.jl:39]: no such file in this PR. forgot `git add`?

**Comment by fieker** [comment]
On Wed, Jan 12, 2022 at 07:12:39AM -0800, Max Horn wrote:
> @fingolfin commented on this pull request.
> 
> 
> 
> >  end
>  
>  include("Rings/binomial-ideals-test.jl")
>  
> -include("Examples/galois-test.jl")
> -include("Examples/ModStdQt-test.jl")
> -include("Examples/ModStdNF-test.jl")
> +include("Experimental/PlaneCurve-test.jl")
> +include("Experimental/galois-test.jl")
> +include("Experimental/gmodule-test.jl")
> 
> no such file in this PR. forgot `git add`?
> 
indeed and amended

**Comment by fingolfin** [comment]
Tests fail with
```
MethodError: no method matching singular_ring(::QabField; keep_ordering=false)
```

File: experimental/GModule/GModule.jl [hunk 2]
```diff
@@ -569,7 +570,7 @@ module RepPc
 using Oscar
 
 Base.pairs(M::MatElem) = Base.pairs(IndexCartesian(), M)
-Base.pairs(::IndexCartesian, M::MatElem) = Base.Pairs(M, CartesianIndices(axes(M)))
+Base.pairs(::IndexCartesian, M::MatElem) = Base.Iterators.Pairs(M, CartesianIndices(axes(M)))
```
Comment (fieker) [inline experimental/GModule/GModule.jl:573]: Why? as far as I can see, they are identical Base.Pairs and Base.Iterators.Pairs

**Pull request merged**